demisto_sdk/commands/common/content/objects/pack_objects/readme/readme.py

```diff
@@ -7,8 +8,26 @@
 
 
 class Readme(TextObject):
-    def __init__(self, path: Union[Path, str]):
+    def __init__(self, path: Union[Path, str], pack_path: Union[Path, str] = ''):
+        self._path = path
+        self._pack_path = pack_path
```

Comment:
no need to pass `pack_path`:
```suggestion
        self._pack_path = os.path.dirname(path)
```